lib/handler/chunked.c

```diff
@@ -78,22 +78,22 @@ static void on_setup_ostream(h2o_filter_t *self, h2o_req_t *req, h2o_ostream_t *
     if (req->is_subrequest)
         goto Next;
 
-    /* do nothing if not HTTP/1.1 or content-length is known */
-    if (req->res.content_length != SIZE_MAX || req->version != 0x101)
+    /* do nothing if not HTTP/1.1 */
+    if (req->version != 0x101)
         goto Next;
```

Comment:
Same as above. I believe that `send_server_timing_trailer` should be cleared when the HTTP version does not permit sending trailers.

Reply:
Regarding HTTP/1.0, you're right. At the same time I think that using HTTP/2 we always can send trailers. So what we have to do here is like the following?

```c
if (req->version >= 0x200)
    goto Next;
if (req->version != 0x101)
    goto NextWithoutTrailers;
```